scripts/qa_crowbarsetup.sh

```diff
@@ -4116,8 +4116,17 @@ function oncontroller_testsetup
     fi
 
     # this file is created by the designate barclamp
-    designate_pools="/etc/desigate/pools.crowbar.yaml"
-    [[ -e $designate_pools ]] && designate-manage pool update --file $designate_pools
+    designate_pools="/etc/designate/pools.crowbar.yaml"
+    if [[ -e $designate_pools ]]; then
+        designate-manage pool update --file $designate_pools
```

Comment:
shouldn't this be already done by the barclamp?

Reply:
the barclamp leaves a sample pool file  `/etc/designate/pools.crowbar.yaml"` for the admin to review.

Why the barclamp doesnot do this ? 
The way designate is currently installed, a zone will be created as soon as a vm is created, the only thing stopping it is "unavailability" of pool in designate. This will have security implication (allowing anyone who can create a vm to create a zone, leaking dns zone information on un-intended network, causing conflicts in dns resolution).

However this is needed to test designate, so adding it here as part of automation.

Reply:
This is being taken care of by https://github.com/crowbar/crowbar-openstack/pull/2192